pkgs/development/python-modules/junos-eznc/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonPackages
+, six
+, scp
+, pyserial
+, paramiko
+, netaddr
+, ncclient
+, lxml
+, jinja2
+, pyyaml
+}:
+
+buildPythonPackage rec {
+  pname = "junos-eznc";
+  version = "2.1.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "95a037cdd05618a189517357e46a06886909a18c7923b628c6ac43d5f54b2912";
+  };
+
+  buildInputs = [
+    scp six pyserial paramiko netaddr ncclient lxml jinja2 pyyaml
```

Comment:
Are all those dependencies only needed at build time of the package? All python dependencies that are needed at runtime, should be also in `propagatedBuildInputs`.